### DISCUSSION TITLE
Questions about addon-detection and ip-based tracking

### DISCUSSION THREAD

**Discussion by dhowe** [dhowe/AdNauseam#2068] [Q&A] [answered]
From  @ChaosNicro:
> Hey there,
> I am considering to put this in both my mobile Fennec and desktop Firefox installations, however there are two things that are keeping me:
> 
> Firstly, while I have taken big steps to eliminate Google from my devices and use webapps and alternative clients where I haven't been able to do that yet, I would like to avoid having my account terminated. Is that still an issue and how does ublock get a pass here, which I have been using for years?
> 
> Secondly, is there any impact on privacy when making the http-requests to adservers? Do things like ip- or useragent-based profiling need to be mitigated separately? I suppose a dynamic ip isn't as vulnerable here but ADN could make fingerprinting much easier, right?
> 
> As a final note: Is there a space for general discussion, like for example a Matrix-Room? It might help with keeping the GitHub focused on development.
> 
> Thanks in advance.

**Comment by dhowe** [comment, marked as answer]
> Firstly, while I have taken big steps to eliminate Google from my devices and use webapps and alternative clients where I haven't been able to do that yet, I would like to avoid having my account terminated. Is that still an issue and how does ublock get a pass here, which I have been using for years?

We haven't had reports of this occurring in quite some time

> Secondly, is there any impact on privacy when making the http-requests to adservers? Do things like ip- or useragent-based profiling need to be mitigated separately? I suppose a dynamic ip isn't as vulnerable here but ADN could make fingerprinting much easier, right?

As the FAQ [here](https://github.com/dhowe/AdNauseam/wiki/FAQ#isnt-it-safer-just-to-use-an-adblocker-why-engage-with-ad-networks-at-all) says:

> While AdNauseam is far safer than using no blocker at all, it is indeed marginally safer for one to simply use a strong adblocker and protect themselves. But it is also safer to stay at home rather than to attend a protest. Using an adblocker does little to change the status quo (especially for those users without the resources to install/configure one, and so remain at risk). AdNauseam, and the obfuscation strategy in general, instead presents a possible avenue for collective resistance; a means of questioning and perhaps eventually, changing the system. But this is not for everyone. If your goal is primarily self-protection, this tool may not be for you...

AdNauseam doesn't do IP obfuscation, but cookies are stripped, the user-agent is anonymized, and the referrer is removed from outgoing ad-click requests.  Thus the browser fingerprint with AdNauseam installed _should_ be less identifying than without (though this would need testing to say for sure).

**Reply by FrostKnight** [reply]
Is hiding ones own IP address even possible on firefox or tor browser at this point? 
I mean, like completely...
I heard tor browser had some issues with firefox's new change to webextensions and still does...
Also, many websites would likely break and/or ban people. 
I would be willing to help your crusade, but I refuse to use modern web browsers, except maybe badwolf, but only if that actually had adblocking and noscripting features.
This all being said, I am fully on board for making these ad companies work for their revenue to such a point, that I want to starve them as much as possible.

**Reply by ChaosNicro** [reply]
You're reviving a zombie-thread here but in theory, your IP is a necessary part of the transport. Even if you could forge the sender-field the ad-server won't likely accept a click where their response can't be delivered, sweeping aside all the issues tcp and all other involved protocols will have with invalid addresses. 
This leaves using a proxy which you can freely do on your own, tor or otherwise, I don't see why it wouldn't work with the add-on.
As always the degree of paranoia to apply depends on your threat-model but assuming your ISP assigns dynamic IPs that periodically change, ad-servers won't likely bother to profile you accross IPs without fingerprinting and cookies when 99% of users can be tracked through those. As I've alluded to. Lastly, as was said above, protest does take choosing the (very slight) discomfort sometimes. 
If you feel like a SWAT-Team may be camping outside your flat this may not be for you.

**Comment by ChaosNicro** [comment]
Also feel free to mark your answer as accepted if you like, @dhowe Since I'm not listed as OP.
